src/app/lists/TooltipCell.tsx

```diff
@@ -0,0 +1,39 @@
+import SpeechBubbleIcon from "@/components/Icons/SpeechBubbleIcon";
+import React, { useState } from "react";
+import { styled, useTheme } from "styled-components";
+
+const CellDiv = styled.div`
+    pointer-events: inherit;
+`;
+
+interface Props {
+    cellValue: string;
+    tooltipValue?: string;
+}
+
+const TooltipCell: React.FC<Props> = (props) => {
+    const [showTooltip, setShowTooltip] = useState(false);
+
+    const theme = useTheme();
+
+    if (!props.tooltipValue) {
+        return <>{props.cellValue}</>;
+    }
+
+    return (
+        <CellDiv
+            onMouseEnter={() => setShowTooltip(true)}
+            onMouseLeave={() => setShowTooltip(false)}
+            onClick={() => setShowTooltip(true)}
+        >
+            {props.cellValue}
+            <SpeechBubbleIcon
+                onHoverText={props.tooltipValue}
+                showTooltip={showTooltip}
+                color={theme.accentBackgroundColor}
```

Comment:
not sure why this is not `accent.background`

Reply:
I've updated this now to reflect the changes made in the new theming system